app/domain/logs.rb

```diff
@@ -339,6 +339,27 @@ module AuthnJwt
         msg: "JWT authentication passed successfully",
         code: "CONJ00066D"
       )
+
+      FetchingJwtClaimsToValidate = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching JWT claims to validate",
+        code: "CONJ00067D"
+      )
+
+      FetchedJwtClaimsToValidate = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetched JWT claims to validate",
+        code: "CONJ00068D"
+      )
+
+      AddingJwtClaimToValidate = ::Util::TrackableLogMessageClass.new(
+        msg: "Adding JWT claim '{0-claim-name}' to validate",
```

Comment:
@shulifink wdyt about this message? the english seems odd.

Reply:
Correct i changed it